test/jdk/sun/security/ssl/SSLLogger/LoggingFormatConsistency.java

```diff
@@ -117,10 +123,14 @@ protected void runServerApplication(SSLSocket socket) throws Exception {
             out.writeBytes("Content-Type: text/html\r\n\r\n");
             out.write(responseBytes);
             out.flush();
-        } catch (IOException e) {
+        } catch (Exception e) {
+            System.err.println("Server-Side Exception: " + e);
+            e.printStackTrace(System.err);
             out.writeBytes("HTTP/1.0 400 " + e.getMessage() + "\r\n");
             out.writeBytes("Content-Type: text/html\r\n\r\n");
             out.flush();
+        } finally {
+            socket.getInputStream().readAllBytes();
```

Comment:
This will cause the server side to block until the client closes the socket. Is that what you really want to do? (It may be - but if the client is a regular HTTP client (HttpURLConnection / HttpClient) it will not close the connection until its keep-alive delay (may be up to 20mins) is expired.

Reply:
I've updated the PR by removing the custom server logic, more detail can be found in my latest comment.

Evan :)